### PR TITLE
Fix metadata return vals

### PR DESCRIPTION
**Pull request by Dazzla**
Metadata return vals fixed, needs release.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,7 @@
 moka
+
+.idea/**/*.xml
```
Comment (jspc) [inline .gitignore]: these should be in your global gitignore, not project specific.

File: json/ingestMetadataDefinition.json [hunk 2]
```diff
@@ -1 +1 @@
...
...T)","default":false,"value":"http://api.ft.com/things/4af53874-c6b3-4969-965a-7cc68bee5b42"},{"id":107587,"name":"ALST","displayName":"World - A list (ALST)","default":false,"value":"http://api.ft.com/things/3b3efaa4-f3d2-4456-a119-857f29c4a404"},{"id":107588,"name":"FFT","displayName":"World - FirstFT (FFT)","default":false,"value":"http://api.ft.com/things/60b1dc20-63e1-4361-968e-5ef73afd9d0c"},{"id":107589,"name":"ARTS","displayName":"Life - FT Arts (ARTS)","default":false,"value":"http://api.ft.com/things/f2e4de5f-c65e-4a8f-a042-e6c4f3b9bcdb"},{"id":107590,"name":"LIFE","displayName":"Life - FT Life (LIFE)","default":false,"value":"http://api.ft.com/things/5cd8e3ff-a8db-4330-a67c-e37c48dbf008"},{"id":107591,"name":"WLTH","displayName":"Life - FT Wealth (WLTH)","default":false,"value":"http://api.ft.com/things/c099c3ec-06a5-4f24-98b5-779dcf65c430"},{"id":107592,"name":"SP","displayName":"Life - Special Projects (SP)","default":false,"value":"http://api.ft.com/things/5c9a6095-11dd-41ac-8b10-07355c0b599f"},{"id":107593,"name":"LAA","displayName":"Life - FT Life &amp; Arts (LAA)","default":false,"value":"http://api.ft.com/things/d743fcb9-9fea-4e4d-a550-8b786b0dda5c"}]}],"rules":[]}
```
Comment (jspc) [inline json/ingestMetadataDefinition.json]: Missing EOL-at-EOF

File: json/publishMetadataDefinition.json [hunk 3]
```diff
@@ -1 +1 @@
...
...idationHandler":"tv.nativ.mio.metadata.variable.def.validation.MaxLengthValidationHandler","valueGeneratorType":null,"unitString":null,"commentable":false,"isVisible":true,"preProcessors":[]},{"id":102735,"name":"brightcove-metadata-tags","displayName":"Brightcove Metadata Tags","description":"{}","type":"complex","multiplicity":"0..*","searchable":true,"editable":true,"required":true,"formType":null,"format":null,"formatDescription":null,"validation":null,"maxLength":-1,"expressionEnabled":false,"validationDescription":null,"validationHandler":null,"valueGeneratorType":null,"unitString":null,"commentable":false,"isVisible":true,"preProcessors":[],"children":[{"id":102736,"name":"brightcove-tag","displayName":"Brightcove Tag","description":"{}","type":"text","multiplicity":"1","searchable":false,"editable":false,"required":false,"formType":"textarea","format":null,"formatDescription":null,"validation":null,"maxLength":-1,"expressionEnabled":false,"validationDescription":null,"validationHandler":"tv.nativ.mio.metadata.variable.def.validation.MaxLengthValidationHandler","valueGeneratorType":null,"unitString":null,"commentable":false,"isVisible":true,"preProcessors":[]}]}],"rules":[]}
```
Comment (jspc) [inline json/publishMetadataDefinition.json]: Ditto Missing EOL-at-EOF

**Review by jspc — CHANGES_REQUESTED** [review]
@Dazzla: please fix errors.

**Comment by Dazzla** [comment]
Fixed

**Comment by jspc** [comment]
LGTM

**Pull request merged**